x/ecocredit/client/tx.go

```diff
@@ -49,19 +61,143 @@ func txCreateClass() *cobra.Command {
 			}
 			b, err := base64.StdEncoding.DecodeString(args[2])
 			if err != nil {
-				return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "base64_metadata is malformed, proper base64 string is required")
+				return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "metadata is malformed, proper base64 string is required")
 			}
 
-			c, ctx, err := mkMsgClient(cmd)
+			c, err := newMsgSrvClient(cmd)
 			if err != nil {
 				return err
 			}
-			_, err = c.CreateClass(cmd.Context(), &ecocredit.MsgCreateClassRequest{
+			msg := ecocredit.MsgCreateClassRequest{
 				Designer: args[0], Issuers: issuers, Metadata: b,
-			})
-			return mkTx(ctx, cmd, err)
+			}
+			_, err = c.client.CreateClass(cmd.Context(), &msg)
+			return c.send(err)
+		},
+	}
+}
+
+func txCreateBatch() *cobra.Command {
+	return &cobra.Command{
+		Use:   "create-batch [issuer] [class_id] [metadata] [issuance]",
+		Short: "Issues a new credit batch",
+		Long: `Issues a new credit batch.
+
+Parameters:
+  issuer:    issuer address
+  class_id:  credit class
+  metadata:  base64 encoded issuance metadata
+  issuance:  YAML encode issuance list. Note: numerical values must be written in strings.
+             eg: '[{recipient: "xrn:sdgkjhs2345u79ghisodg", tradableUnits: "10", retiredUnits: "2"}]'
```

Comment:
```suggestion
             eg: '[{recipient: "xrn:sdgkjhs2345u79ghisodg", tradableunits: "10", retiredunits: "2"}]'
```
Fixing to accurately reflect what the `yaml.Unmarshal()` expects